CODING_STANDARDS.md

```diff
@@ -354,3 +425,14 @@ When it is not super obvious, include a brief description of what a class repres
 ```
 
 [ts-mixins]: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#support-for-mix-in-classes
+
+#### Prefer CSS classes to tag names for styling
+Targeting tag names can cause conflicts with the MDC version of the component. For this reason, use
+css class names defined by us instead of tag names.
+```scss
+/** Do: */
+.mat-mdc-slider { ... }
+
+/** Don't: */
+mdc-slider { ... }
+```
```

Comment:
We also prefer classes over attributes for consistency

Reply:
@jelbourn Do you want me to add exactly this to the description?

```
Targeting tag names can cause conflicts with the MDC version of the component. For this reason, use
CSS class names defined by us instead of tag names. We also prefer classes over attributes for
consistency.
```

Reply:
Sure, that works